packages/moonstone/Picker/Picker.js

```diff
@@ -203,7 +203,7 @@ const PickerBase = kind({
 			if (voiceLabelsExt) {
 				voiceLabel = voiceLabelsExt;
 			} else {
-				voiceLabel = children.map((child) => ((typeof child === 'number' || typeof child === 'string') ? child : ''));
+				voiceLabel = children && Array.isArray(children) && React.Children.map(children, (child) => ((typeof child === 'number' || typeof child === 'string') ? child : ''));
```

Comment:
It's not necessary to do both checks `React.Children.map`, I believe.  I'll fix this up.